README.md

```diff
@@ -0,0 +1,109 @@
+<a id="top"></a>
+
+# peAk
+
+```
+Code Fellows; seattle-ja-401d2
+Darrin Howell, Nick Crain, Jason Burns
+```
+
+### Target Demographic:
+Avid snowboarders with robust social lives who want to streamline their mountian plans, together.
+
+### Preliminary Vision:
+The app that we plan to develop is called “Ride Together”. This app will make API requests to get weather reports of ski resorts within a specified radius of their phone’s location (NOAA API / Cordova API). In addition to making requests for weather, the app will render the nearby ski resorts (Google Maps API), and show the icons of fellow app users who have checked in to that resort for a trip. After rendering the maps page, users will have the ability to send invites to other friends / groups of friends alerting them (Twilio API) to join them on their snowboarding trip. We will have a database of user checkins as well as group meetups that specify time, people included, and mountain visited. We could also add in a search function that would allow users to check-in / plan trips at mountains all over the world. 

```

Comment:
Surprising that this is still in the future tense! Once you've begun development, it's more traditional to update this to reflect that the app is built.